conf/log4j.properties

```diff
@@ -11,7 +11,7 @@
 
 # May get overridden by System Property
 
-log4j.rootLogger=INFO, ${alluxio.logger.type}, ${alluxio.remote.logger.type}
+log4j.rootLogger=INFO, Console, ${alluxio.logger.type}, ${alluxio.remote.logger.type},EventCounter
```

Comment:
What user-facing changes will be introduced after changing this line?
I am a little worried about changing the rootLogger

Reply:
There is no change on the client side. This line of change can make the log output to the eventcounter class

Reply:
I suggest to remove Console logger here, if who want to add this logger, just define `alluxio.logger.type`.

For `EventCounter `, an extra space between comma and letter E is necessary to keep style consistent.